pkg/scorecard.go

```diff
@@ -176,3 +196,52 @@ func RunScorecard(ctx context.Context,
 	}
 	return ret, nil
 }
+
+// RunScorecard runs enabled Scorecard checks on a Repo.
+func RunScorecard(ctx context.Context,
+	repo clients.Repo,
+	commitSHA string,
+	commitDepth int,
+	checksToRun checker.CheckNameToFnMap,
+	repoClient clients.RepoClient,
+	ossFuzzRepoClient clients.RepoClient,
+	ciiClient clients.CIIBestPracticesClient,
+	vulnsClient clients.VulnerabilitiesClient,
+) (ScorecardResult, error) {
+	return runScorecard(ctx,
+		repo,
+		commitSHA,
+		commitDepth,
+		checksToRun,
+		[]string{},
+		repoClient,
+		ossFuzzRepoClient,
+		ciiClient,
+		vulnsClient,
+	)
+}
+
+// RunScorecard runs enabled Scorecard checks on a Repo.
```

Comment:
Can we have the comment reflect the right function name, and have more of a disclaimer?
```
// ExperimentalRunProbes is experimental. Do not depend on it, it may be removed at any point.
```

Reply:
Done in https://github.com/ossf/scorecard/pull/3688/commits/e005d078ca48757e571e978139631b633de4694e